src/pages/ApplicationList/components/EditPlatFormUserDialog/index.tsx

```diff
@@ -0,0 +1,244 @@
+                    disabled={true}
+                    {...init('name', {
+                      rules: [
+                        {
+                          required: true,
+                          pattern: checkName,
+                          message: <Translation>Please enter a valid name</Translation>,
+                        },
+                      ],
+                    })}
+                  />
+                </FormItem>
+              </Col>
+              <Col span={12} style={{ padding: '0 8px' }}>
+                <FormItem label={<Translation>Alias</Translation>}>
+                  <Input
+                    name="alias"
+                    placeholder={i18n.t('Please input the alias').toString()}
+                    {...init('alias', {
+                      rules: [
+                        {
+                          minLength: 2,
+                          maxLength: 64,
+                          message: 'Enter a string of 2 to 64 characters',
+                        },
+                      ],
+                    })}
+                  />
+                </FormItem>
+              </Col>
+            </Row>
+            <Row>
+              <Col span={12} style={{ padding: '0 8px' }}>
+                <FormItem label={<Translation>Password</Translation>} required>
+                  <Input
+                    name="password"
+                    htmlType="password"
+                    placeholder={i18n.t('Please input the password').toString()}
+                    {...init('password', {
+                      rules: [
+                        {
+                          required: true,
+                          pattern: checkUserPassword,
+                          message: (
+                            <Translation>
+                              Password should be 8-16 bits and contain at least one number and one
+                              letter
+                            </Translation>
+                          ),
+                        },
+                        {
+                          validator: (rule, value, callback) => {
+                            const { password, ConfirmPassword } = this.field.getValues();
+                            if (password && ConfirmPassword && password !== ConfirmPassword) {
+                              callback(i18n.t('Inconsistent password entry twice'));
+                            } else {
+                              this.field.setError('password');
+                              this.field.setError('ConfirmPassword');
+                              callback();
+                            }
+                          },
+                        },
+                      ],
+                    })}
+                  />
+                </FormItem>
+              </Col>
+
+              <Col span={12} style={{ padding: '0 8px' }}>
+                <FormItem label={<Translation>ConfirmPassword</Translation>} required>
```

Comment:
For consistency, we do not set this field, but provide a button with an eye icon for showing the password.

Reply:
done.remove it ,and implement eye icon